troupon/static/scss/partials/_sections.scss

```diff
@@ -104,6 +104,13 @@ body {
 
   .member-options {
     z-index: z-index(dropdown);
+    .user-info, .dropdown-header {

```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own single line